### PR TITLE
*: format imports and dependencies

### PR DESCRIPTION
**Pull request by Connor1996**
This PR does:
- format the imports in order by `rustfmt`
- format the dependencies in order by `cargo sort`

### CODE REVIEW

File: rust-toolchain [hunk 1]
```diff
@@ -0,0 +1 @@
+nightly-2022-02-14
```
Comment (BusyJay) [inline rust-toolchain:1]: This is a bad idea. A library should not bind to a specific version.

File: rustfmt.toml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+version = "Two"
```
Comment (BusyJay) [inline rustfmt.toml:1]: What's wrong with the default format?

File: rust-toolchain (hunk 1 above)
Comment (Connor1996) [reply rust-toolchain:1]: ok, remove it

File: rustfmt.toml (hunk 2 above)
Comment (Connor1996) [reply rustfmt.toml:1]: just be consistent with tikv, not sure why change to version two before

Comment (BusyJay) [reply rustfmt.toml:1]: I suggest to just use all the default settings of rustfmt. No one is going to sync the settings across repos when there is further change.

Comment (Connor1996) [reply rustfmt.toml:1]: no need to sync the changes, it's just a initial to make the imports in order. We can change it further on demand regardless of how TiKV does.  Seems version two is a good practice, see https://github.com/rust-lang/rust/blob/master/rustfmt.toml#L2

Comment (BusyJay) [reply rustfmt.toml:1]: > ..regardless of how TiKV does..

I thought you wanted "be consistent with tikv".

What do you mean "make the imports in order"?

Comment (Connor1996) [reply rustfmt.toml:1]: `use xxx` is sorted in order

**Pull request merged**